pkg/replication/admission.go

```diff
@@ -37,6 +38,9 @@ import (
 )
 
 const (
+	// User account this component uses
```

Comment:
"this component"? replication admission is a separate component from replication controller...
I would rephrase comment to explicitly mention replication controller.

Reply:
I view replication as an entire subsystem. The implementation is split across a controller and a couple of webhooks. I take it you see it differently?

Reply:
Fair enough, but in this case it's replication controller request to API server being intercepted by a webhook, so it seems useful to be more clear about what we do.